.travis.yml

```diff
@@ -69,7 +81,7 @@ matrix:
     stage: acceptance
   - rvm: 2.5.7
     services: docker
-    env: BEAKER_set="centos-7" BEAKER_PUPPET_COLLECTION=puppet6 BEAKER_sensu_mode=full
```

Comment:
why do full runs with puppet versions 5 and 7 and not 6?

Reply:
I think that was copy + paste mistake in vim.  I put the puppet6 tests back.